slip132/src/lib.rs

```diff
@@ -551,7 +542,7 @@ impl VersionResolver for DefaultResolver {
     fn application(kv: &KeyVersion) -> Option<Self::Application> {
         match kv.as_bytes() {
             &VERSION_MAGIC_XPUB | &VERSION_MAGIC_XPRV | &VERSION_MAGIC_TPUB
-            | &VERSION_MAGIC_TPRV => None,
+            | &VERSION_MAGIC_TPRV => Some(KeyApplication::Hashed),
```

Comment:
The idea with returning `None` here was the fact that Xpub's and related test types are allowed to be used in descriptors in _any_ context, not just pre-segwit

Reply:
Makes sense, thanks for clarifying and sorry for the late reply. Maybe we should add a note to the docs then. Right now it says `Returns None if the version is not recognized/unknown to the resolver.` Using this library I'd be confused why xpub related types are unknown